sw/device/silicon_creator/lib/cert/dice_chain.c

```diff
@@ -237,6 +251,22 @@ static rom_error_t dice_chain_load_flash(
   return kErrorOk;
 }
 
+// Add the hash digest to the last of the page.
+static rom_error_t dice_chain_seal_page(void) {
+  // Raise an error if the remaining space in the tail buffer is insufficient
+  // for the hash digest.
+  if (dice_chain_get_tail_size() < sizeof(hmac_digest_t)) {
+    return kErrorDicePageExhausted;
+  }
```

Comment:
If we change to a struct that holds the cert data and digest, I think we can also eliminate this error check because the other tail handling functions are going to be bounded by `sizeof(cert_page::data)`.